indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisor.java

```diff
@@ -3658,6 +3665,25 @@ private boolean checkOffsetAvailability(
            && makeSequenceNumber(earliestOffset).compareTo(makeSequenceNumber(offsetFromMetadata)) <= 0;
   }
 
+  protected void emitNoticesQueueSize()
+  {
+    if (spec.isSuspended() || !stateManager.isSteadyState()) {
+      // don't emit metrics if supervisor is suspended or not in a healthy running state
+      // (lag should still available in status report)
```

Comment:
why do we mention lag here?

Reply:
seems to be copy-pasta from emitLag()

Reply:
would it still make sense to keep emitting queue size if the supervisor is not in steady state? It would still have to process all the notices in the queue once it resumes or becomes healthy again, no?

Reply:
>seems to be copy-pasta from emitLag()
That's right. I'll remove it.

> would it still make sense to keep emitting queue size if the supervisor is not in steady state?
that sounds reasonable. I was just trying to follow the same pattern that was used for emitLag. Will update the code accordingly.